sites.yml

```
@@ -1538,6 +1538,15 @@ sites:
     maintainers:
       - "[Ivan G Torre](https://imagej.net/User:Ivangtorre)"
 
+  - name: "MultiStackReg"
+    id: "MultiStackReg"
+    url: "https://sites.imagej.net/MultiStackReg/"
+    description: >-
+      [MultiStackReg](https://sites.imagej.net/MultiStackReg/)
```

Instead of pointing to the hosted files, the description should rather point to documentation of the plugin. I suggest creating https://imagej.net/plugins/multistackreg and pointing to it here.

```suggestion
      [MultiStackReg](https://imagej.net/plugins/multistackreg)
```